### DISCUSSION TITLE
Attended Access prompt not working when at loggin screen

### DISCUSSION THREAD

**Discussion by bruno-paduin** [immense/Remotely#920] [Q&A]
Hello, everyone.
I'm implementing Remotely in the company I work for and I'm very impressive with the simplicity of the server and agent installation/deployment.
The only thing that stopped me on going forward is that we have an internal policy where all remote access must be attended, but when the user cannot login to their computer and requests us for a remote support (in the Ctrl+Alt+Del screeen), the only way we can do it is by accessing the Remotely "Server Config" and disabling "Enforce Attended Access". When our support is finished, we re-enable that option again. It works, but it is not optimal.

Is it possible to make the "Attended Access Prompt" appear even when in the Ctrl+Alt+Del screen?

I'm currently using the 2024.07.16.1109

Thank you.

**Comment by bitbound** [comment]
I've opened a PR for this:  #924 

It will be included in the next Docker image.

**Comment by bitbound** [comment]
This has been released:  https://github.com/immense/Remotely/releases/tag/v2024.08.07.0819
